packages/grafana-data/src/field/overrides/processors.ts

```diff
@@ -69,8 +69,8 @@ export const stringOverrideProcessor = (
   context: FieldOverrideContext,
   settings?: StringFieldConfigSettings
 ) => {
-  if (value === null || value === undefined) {
-    return value;
+  if (value === null || value === undefined || value === '') {
+    return null;
```

Comment:
this changes the logic a bit, not sure what is correct to return here, null or undefined